### PR TITLE
🔧 add npm test

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -12,5 +12,4 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ljharb/actions/node/install@main
         name: 'nvm install lts/* && npm install'
-      - run: npm run lint
-
+      - run: npm test
```
Comment (ljharb) [inline .github/workflows/lint.yml:15]: i would still expect the lint workflow to run the lint script

Comment (ctcpip) [reply .github/workflows/lint.yml:15]: if I rename the workflow, does that satisfy?

Comment (ljharb) [reply .github/workflows/lint.yml:15]: i mean i guess - what's the benefit tho?

(i'm on board with making `npm test` run all the things locally, but you almost never want that in a single CI job)

Comment (ctcpip) [reply .github/workflows/lint.yml]: ok

**Review by ljharb — APPROVED** [review]

**Pull request merged**